.gitignore

```diff
@@ -75,3 +75,5 @@ cscope.*
 *.swp
 *~
 
+# Visual Studio Code
+.vscode/
```

Comment:
Editing the files with VSC creates several files below this subfolder. Without this ignore, git always want to commit hose files to the repository.